### PR TITLE
Dashboard page

### PR DESCRIPTION
**Pull request by abhinav-anshul**
This PR adds :

1. Dashboard Page to the Arbchain Project
2. Icons specific to the Design board such as Notification icon, Brand Logo(Arbchain Logo)
3. Dummy Chart as a Component using a library called Chart.js

This PR fixes : 
1. Header Avatar logo issues

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -26,6 +27,7 @@ function App() {
         <ActivityProvider>
           <Switch>
             <Route exact path='/login' component={LoginPage} />
```
Comment (koshikraj) [inline src/App.js:31]: The `root` path seems to be still missing.

File: src/components/Account/AccountButton.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
+/* eslint-disable */
 import React from 'react'
 import PropTypes from 'prop-types'
 import { EthIdenticon, GU, RADIUS, textStyle, useTheme } from '@aragon/ui'
 
 import { useWallet } from 'use-wallet'
-import HeaderModule from '../Header/HeaderModule'
+import HeaderModule from '../HeaderREDUNDANT/HeaderModule'
```
Comment (koshikraj) [inline src/components/Account/AccountButton.js:7]: Can you rename this module directory (`HeaderREDUNDANT`) and remove any unnecessary file.

File: src/components/Dashboard/ChartComponentNew.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+class ChartComponentNew extends React.Component {
+  constructor(props) {
+    super(props)
+  }
+  componentDidMount() {
+    const node = this.node
+
+    var myChart = new Chart(node, {
+      type: 'bar',
```
Comment (koshikraj) [inline src/components/Dashboard/ChartComponentNew.js:13]: What are the different types of plottable charts?

File: src/components/Dashboard/DashboardMain.js [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+          margin-top: 0rem;
+          margin-left: 1rem;
+          margin-right: 1rem;
+        `}
+      >
+        Dashboard
+      </div>
+
+      <div
+        style={{
```
Comment (koshikraj) [inline src/components/Dashboard/DashboardMain.js:48]: Shall we maintain most of the CSS separately?

**Comment by koshikraj** [comment]
It's a pretty good PR. General suggestions:

* Use our [branch naming convention](https://www.notion.so/consensolabs/Development-Lifecycle-47942e4ae35b45a79ae4525937382cf3) in the future.
* Better way to maintain CSS as there are a lot of inline CSS
* Try to merge this branch with master and resolve conflict.

File: src/App.js (hunk 1 above)
Comment (abhinav-anshul) [reply src/App.js:31]: Is this why there is a merge conflict?

File: src/components/Account/AccountButton.js (hunk 2 above)
Comment (abhinav-anshul) [reply src/components/Account/AccountButton.js:7]: Yes sure, will do this when I push code for refactoring

File: src/components/Dashboard/ChartComponentNew.js (hunk 3 above)
Comment (abhinav-anshul) [reply src/components/Dashboard/ChartComponentNew.js:13]: These are bar, line, pie, Radar, Bubble, Scatter, Doughnut. For in-depth reference check out [Types of Charts](https://www.chartjs.org/docs/latest/charts/)

File: src/components/Dashboard/DashboardMain.js (hunk 4 above)
Comment (abhinav-anshul) [reply src/components/Dashboard/DashboardMain.js:48]: Yes, perhaps we have to go in that direction eventually.

File: src/App.js (hunk 1 above)
Comment (koshikraj) [reply src/App.js:31]: Maybe. Try to do a merge with `master`, you will figure it out.

File: src/components/Dashboard/ChartComponentNew.js (hunk 3 above)
Comment (koshikraj) [reply src/components/Dashboard/ChartComponentNew.js:13]: For us, line chart is more accurate in this case. Maybe you can replace it with that.

**Comment by koshikraj** [comment]
@abhinav-anshul are you working on the same branch for the arbitration page? If not, we should merge this PR.
Also, merge with the `master` branch to resolve this conflict. Let me know if you are facing any issues.

**Comment by koshikraj** [comment]
@abhinav-anshul I'm merging this PR as I need the latest code to work on the client integration. Please take care of converting the `buttons` of the `panel` to plain `divs` (A separate component) similar to the mockup. I will expect this change in the other PR.

**Pull request merged**